server/utils/hasProperty.js

```diff
@@ -0,0 +1,16 @@
+function hasValidProperty(name){
+    return name.hasOwnProperty('valid');
```

Comment:
Same for these, the response from mongoDB should be enough to validate the results

Reply:
For this change, I have to change the design. I am going to pass all my (req,res) param to controllers. It will handle all the errors with its status.